README.md

```diff
@@ -84,7 +84,7 @@ can enjoy lh-tag automagic update of the database, and improved tag selection.
    * or where `.svn/` is found in parent directories ;
    * or asked to the end-user (previous values are recorded in case several
      files from a same project are opened).
- * `lh#tags#add_indexed_ft()`  
```

Comment:
Actually these trailing whitespaces where there on purpose: to introduce a newline within a bullet point entry.